package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/doctrine.js",
   "version": "2.1.0",
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=6.0.0"
```

Comment:
Since we don't support Node 7, for example, would it make sense to make this line similar to that for ESLint?

https://github.com/eslint/eslint/blob/3dd7493a1e6a345ff82d9e5ed5ca797395f349bb/package.json#L126

Reply:
I'm not sure it matters in this context. If we are going to deprecate the entire module, might as well keep things simple.

Reply:
Good point!